python/tvm/script/tir/special_stmt.py

```diff
@@ -17,6 +17,7 @@
 """TVM Script Parser Special Stmt Classes"""
 # pylint: disable=unused-argument, no-self-argument, inconsistent-return-statements
 # pylint: disable=relative-beyond-top-level
+from ctypes import ArgumentError
```

Comment:
Probably we don't need this particular kind of error from ctypes. What about we just raise ValueError?